item.rb

```diff
@@ -1,19 +1,20 @@
-require_relative 'date_handler'
-
+require_relative './date_handler'
 class Item
-  attr_accessor :publish_date, :genre
-  attr_reader :id, :source, :author, :label, :archived
+  attr_accessor :publish_date
+  attr_reader :id, :genre, :source, :label, :archived, :author
```

Comment:
You will need to keep :genre, :source, :label and :author in attr_accessor since we are using them in the add_item methods in the respective classes